content/en/news/releases/1.21.x/announcing-1.21/change-notes/index.md

```diff
@@ -138,8 +157,36 @@ dynamic metadata key.
 
 - **Updated** the default value of the feature flag `VERIFY_CERT_AT_CLIENT` to `true`.
 This means server certificates will be automatically verified using the OS CA certificates when not using a `DestinationRule` `caCertificates` field.
-If undesired, please use the new `compatibilityVersion` feature to fallback to old behavior, or `insecureSkipVerify`
-field in `DestinationRule` to skip the verification.
+If undesired, please use the new `compatibilityVersion` feature
+to fallback to old behavior, or `insecureSkipVerify` field
+in `DestinationRule` to skip the verification.
+
+- **Added** an environment variable `COMPLIANCE_POLICY` to
+Istio components for enforcing TLS restriction for compliance
+with FIPS. When set to `fips-140-2` on the Istiod container,
+the Istio Proxy container, and all other Istio components,
+TLS version is restricted to `v1.2`, the cipher suites to a
+subset of `ECDHE-ECDSA-AES128-GCM-SHA256`,`ECDHE-RSA-AES128-GCM-SHA256`,
+`ECDHE-ECDSA-AES256-GCM-SHA384`, `ECDHE-RSA-AES256-GCM-SHA384`,
+and ECDH curves to `P-256`.
+
+These restrictions apply on the following data paths:
+* Google gRPC side requests from Envoy proxies
+  (e.g. Stackdriver extensions).
+* Istiod injection and validation webhook servers.
+* Istiod xDS server.
+* mTLS communication between Envoy proxies.
+* regular TLS on the downstream and the upstream of Envoy proxies
+    (e.g. gateway)
+
+The restrictions are not applied on the following data paths:
+* Istiod to Kubernetes API server.
+* JWK fetch from Istiod.
+* Wasm image and URL fetch from Istio Proxy containers.
+* ztunnel.
+Note that Istio injector will propagate the value
+of `COMPLIANCE_POLICY` to the injected proxy container, when set.
+([Issue #49081](https://github.com/istio/istio/issues/49081))
```

Comment:
```suggestion
    These restrictions apply on the following data paths:
    * Google gRPC side requests from Envoy proxies
  (e.g. Stackdriver extensions).
    * Istiod injection and validation webhook servers.
    * Istiod xDS server.
    * mTLS communication between Envoy proxies.
    * regular TLS on the downstream and the upstream of Envoy proxies
    (e.g. gateway)

    The restrictions are not applied on the following data paths:
    * Istiod to Kubernetes API server.
    * JWK fetch from Istiod.
    * Wasm image and URL fetch from Istio Proxy containers.
    * ztunnel.

    Note that Istio injector will propagate the value of `COMPLIANCE_POLICY` to the injected proxy container, when set. ([Issue #49081](https://github.com/istio/istio/issues/49081))
```

(This might be too many spaces indent for Hugo?)

Reply:
Looks good from the preview https://deploy-preview-14717--preliminary-istio.netlify.app/latest/news/releases/1.21.x/announcing-1.21/change-notes/